### PR TITLE
Update Razor SDK packing logic to not include Pack at properties layer.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- We now rely on the Razor source inputs layer to add packing to cshtml content items.

#2378 
#2384 

@mkArtakMSFT This is ready to go, let me know when i'm good to merge.

### CODE REVIEW

File: test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/PackIntegrationTest.cs [hunk 1]
```diff
@@ -15,6 +15,46 @@ public PackIntegrationTest(BuildServerTestFixture buildServer)
         {
         }
 
+        [Fact]
+        [InitializeTestProject("ClassLibrary")]
+        public async Task Pack__NoBuild_Works_IncludesRazorAssembly()
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/PackIntegrationTest.cs]: nit: two underscores after `Pack`

Comment (pranavkm) [reply test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/PackIntegrationTest.cs]: Could you also add a test that verifies we do add the files in no-build scenarios if the flag is set?

File: test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/PackIntegrationTest.cs [hunk 2]
```diff
@@ -15,6 +15,67 @@ public PackIntegrationTest(BuildServerTestFixture buildServer)
...
+            var result = await DotnetMSBuild("Build");
+            Assert.BuildPassed(result);
+
+            result = await DotnetMSBuild("Pack", "/p:NoBuild=true /p:IncludeRazorContentInPack=true");
+            Assert.BuildPassed(result);
+
+            Assert.NuspecContains(
+                result,
+                Path.Combine("obj", Configuration, "ClassLibrary.1.0.0.nuspec"),
+                @"<files include=""any/netstandard2.0/Views/Shared/_Layout.cshtml"" buildAction=""Content"" />");
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/PackIntegrationTest.cs]: I'm assuming this will fail on travis but we'll see.

**Comment by NTaylorMullen** [comment]
🆙 📅

**Comment by pranavkm** [comment]
@NTaylorMullen this should be good to go in now.

**Pull request merged**